### PR TITLE
Issue438 cooler

### PR DESCRIPTION
**Pull request by MichaMans**
Could we merge this to enable the cooler? 
@PRemmen @DaJansenGit do you have additions or objections?

### CODE REVIEW

**Comment by MichaMans** [comment]
@PRemmen if we merge #541 first, the checks should run and i looking forward to your review 😃

File: teaser/data/output/modelicatemplate/AixLib/AixLib_ThermalZoneRecord_TwoElement [hunk 1]
```diff
@@ -73,12 +73,12 @@ record ${zone.parent.name}_${zone.name} "${zone.parent.name}_${zone.name}"
...
+    TNHeat = 50,
     HeaterOn = true,
     hCool = 0,
     lCool = ${zone.model_attr.cool_load},
-    KRCool = 10000,
-    TNCool = 1,
-    CoolerOn = false);
+    KRCool = 100,
+    TNCool = 50,
+    CoolerOn = true);
```
Comment (PRemmen) [inline teaser/data/output/modelicatemplate/AixLib/AixLib_ThermalZoneRecord_TwoElement]: If cooler is on by default, and we have these PI settings, do we expect changes in the heating demand? If so: should we introduce a parameter to set cooler on or off?

**Comment by PRemmen** [comment]
@MichaMans CI fails currently, I'll try to have a detailed look into that issue tomorrow and hopefully this will then be ready to merge :)

**Comment by MichaMans** [comment]
I'll update the branch and we'll see if ci works then

**Comment by MichaMans** [comment]
@PRemmen ci working again :)

**Comment by PRemmen** [comment]
This issue will be handeld in issue510_usecond

**Pull request closed without merging**